javascript/javascript3/week2/index1.html

```diff
@@ -0,0 +1,26 @@
+<head>
+  <title>HyfBay</title>
+  <link href="main.css" rel="stylesheet" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <ul class="marks">
+      <li></li>
+      <li></li>
+      <li></li>
+    </ul>
+
+    <ul class="targets">
+      <li>Red target</li>
+      <li>Blue target</li>
+      <li>Green target</li>
+    </ul>
+    <span>👍</span>
```

Comment:
I am not sure if this emoji can be rendered correctly in all browsers. Either use fontawesome.com or an SVG file. Consider maybe it is not possible if you want to change the color or size of that.